tests/README.md

```diff
@@ -0,0 +1,25 @@
+This directory contains two test files that use pytest.
+
+`test_build.py` checks that the static HTML output of the build process conforms
+to various expectations. It builds static HTML pages based on configurations in
+the `sites/` directory. The tests run various assertions on the static HTML
+output, including snapshot comparisons with previously compiled outputs that are
+stored in `test_build/`. In other words, it uses
+[`pytest-regressions`](https://pytest-regressions.readthedocs.io/) to compare
+the output created during the test run with a previous known and verified output
+(stored under `test_build`) to make sure nothing has changed.
+
+`test_accessibility.py` checks PyData Sphinx Theme components for accessibility
+issues. It's important to note that [only a fraction of accessibility issues can
+be caught with automated
+testing](https://accessibility.blog.gov.uk/2017/02/24/what-we-found-when-we-tested-tools-on-the-worlds-least-accessible-webpage/).
+In contrast to `test_build`, `test_accessibility` checks components as they
+appear in the browser, meaning with any CSS and JavaScript applied. It does this
+by building the PyData Sphinx Theme docs, launching a local server to the docs,
+then visiting the "Kitchen Sink" example pages with
+[Playwright](https://playwright.dev), a program for developers that allows
+loading and manipulating pages with various browsers, such as Chrome (chromium),
+Firefox (gecko), Safari (WebKit). The ["Kitchen Sink"
+examples](https://pydata-sphinx-theme.readthedocs.io/en/stable/examples/kitchen-sink/index.html)
+are a section of the PyData Sphinx Theme docs that showcase the different visual
+components of the PyData Sphinx Theme, such as admonitions, lists, and headings.
```

Comment:
see https://pydata-sphinx-theme.readthedocs.io/en/stable/community/topics/kitchen-sink.html (it explains that the kitchen sink pages are not really "ours", we copy them from elsewhere)